pages/forgotPassword.tsx

```diff
@@ -0,0 +1,11 @@
+import FindYourAccount from '../Components/FindYourAccount/FindYourAccount'
+import Header from '../Components/Header/Header'
+
+export default function forgotPassword () {
```

Comment:
try to keep same format as previous components
```suggestion
const forgotPassword = () => {

and export default forgotPassword at the bottom
```